otter/test/models/test_cass_models.py

```diff
@@ -796,58 +812,60 @@ def modifier(_group, _state):
         self.connection.execute.assert_called_once_with(
             expectedCql, expectedData, ConsistencyLevel.QUORUM)
 
-        self.kz_client.Lock.assert_called_once_with(
-            '/locks/' + self.group.uuid)
-
-        self.lock._acquire.assert_called_once_with(timeout=10)
-        self.lock.release.assert_called_once_with()
+        self.assertEqual(self.lock.path, '/locks/' + self.group.uuid)
+        self.assertFalse(self.lock.acquired)
 
     def test_modify_state_local_lock_before_kz_lock(self):
         """
         ``modify_state`` first acquires local lock then acquires kz lock
         """
+        modifier_d = defer.Deferred()
+        group_state = GroupState(tenant_id=self.tenant_id,
+                                 group_id=self.group_id,
+                                 group_name='a',
+                                 active={},
+                                 pending={},
+                                 group_touched=None,
+                                 policy_touched={},
+                                 paused=True,
+                                 status=ScalingGroupStatus.ACTIVE,
+                                 desired=5)
+
         def modifier(_group, _state):
-            group_state = GroupState(tenant_id=self.tenant_id,
-                                     group_id=self.group_id,
-                                     group_name='a',
-                                     active={},
-                                     pending={},
-                                     group_touched=None,
-                                     policy_touched={},
-                                     paused=True,
-                                     status=ScalingGroupStatus.ACTIVE,
-                                     desired=5)
-            return group_state
+            return modifier_d
 
         self.group.view_state = mock.Mock(return_value=defer.succeed('state'))
-        # setup local lock
+
         llock = defer.DeferredLock()
-        self.group.local_locks = mock.Mock(
-            get_lock=mock.Mock(return_value=llock))
+
+        class Locks(object):
```

Comment:
It's a stub for [`WeakLocks`](https://github.com/rackerlabs/otter/blob/master/otter/util/weaklocks.py)